dask_planner/src/expression.rs

```diff
@@ -653,6 +653,30 @@ impl PyExpr {
             ))),
         }
     }
+
+    /// Returns if a sort expressions is an ascending sort
+    #[pyo3(name = "isSortAscending")]
+    pub fn is_sort_ascending(&self) -> PyResult<bool> {
+        match &self.expr {
+            Expr::Sort { asc, .. } => Ok(asc.clone()),
+            _ => Err(py_type_err(format!(
+                "Provided Expr {:?} is not a sort type",
+                &self.expr
+            ))),
+        }
+    }
+
+    /// Returns if nulls should be placed first in a sort expression
+    #[pyo3(name = "isSortNullsFirst")]
+    pub fn is_sort_nulls_first(&self) -> PyResult<bool> {
+        match &self.expr {
+            Expr::Sort { nulls_first, .. } => Ok(nulls_first.clone()),
```

Comment:
Same as above. [Docs](https://docs.rs/datafusion/latest/datafusion/logical_plan/enum.Expr.html#variant.Sort) show as Scalar so shouldn't require a `.clone()` since it is present on the function local stack. Could use `*asc`